DxDispatch/src/model/Model.h

```diff
@@ -10,6 +10,12 @@
 #include <DirectML.h>
 #include "BucketAllocator.h"
 
+enum class DmlCompileType 
```

Comment:
nit: Feels like this should be in the Model namespace like the other enums/structs since it's specific to the Model::DmlDispatchableDesc.